project/Build.scala

```diff
@@ -414,12 +414,6 @@ object DottyBuild extends Build {
         "org.scala-sbt" % "api" % sbtVersion.value % "test",
         "org.specs2" %% "specs2" % "2.3.11" % "test"
       ),
-      version := {
-        if (isNightly)
-          "0.1.1-" + VersionUtil.commitDate + "-" + VersionUtil.gitHash + "-NIGHTLY"
-        else
-          "0.1.1-SNAPSHOT"
-      },
```

Comment:
This is all done in the global settings: https://github.com/dotty-staging/dotty/blob/0bca6b76ee6c7c9555707d03551593dd04b5fe81/project/Build.scala#L49